yodeploy/virtualenv.py

```diff
@@ -118,12 +150,31 @@ def check_requirements(ve_dir):
         sys.exit(1)
 
 
-def relocateable_ve(ve_dir):
+def relocateable_ve(ve_dir, python_version):
+    # We can only do this for Python 2 under Python 2
```

Comment:
Is this comment for the `relocateable_ve` function as a whole or the code block below?
It doesn't seem to match up with the below code block but I also might not be understanding what is happening.